server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/EverythingCompactionStrategy.java

```diff
@@ -27,7 +27,7 @@
 
   @Override
   public boolean shouldCompact(MajorCompactionRequest request) throws IOException {
```

Comment:
`MajorCompactionRequest` could be modified to add a `getCompactionIterators()` method, then this would have all of the info needed to make a decision.  This might clean the code up a little bit in other places.  This is just something I thought of while looking at this, not something I think has to be done.